src/main/java/com/slimebot/util/Util.java

```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
 			return false;
 		}
 	}
+
+	public static Instant timestamp() {
+		return new Date().toInstant().atZone(ZoneId.systemDefault()).toInstant();
+	}
+
+	public static String padRight(String s, int n) {
+		return padRight(s, ' ', n);
+	}
+
+	public static String padLeft(String s, int n) {
+		return padLeft(s, ' ', n);
+	}
+
+	public static String padRight(String s, char padChar, int n) {
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.append(padChar);
+		return sb.toString();
+	}
+
+	public static String padLeft(String s, char padChar, int n) {
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.insert(0, padChar);
+		return sb.toString();
```

Comment:
return padString.repeat(n) + s

Reply:
no, for the same reason as in my other comment